src/scss/_back-to-top.scss

```diff
@@ -0,0 +1,36 @@
+#return-to-top {
+  position: fixed;
+  bottom: 1.25em; //20px
+  right: 1.25em; //20px
+  background: rgb(0, 0, 0);
```

Comment:
Sure, go for it! I forked a pen on codepen, so here's how I originally did the smooth scrolling: http://codepen.io/rachelbroering/pen/XKxYAA. I'll also make sure to check out that button. Would you recommend any other changes?